metricbeat/docs/modules/aws/sqs.asciidoc

```diff
@@ -3,7 +3,7 @@ This file is generated! See scripts/mage/docs_collector.go
 ////
 
 [[metricbeat-metricset-aws-sqs]]
-=== aws sqs metricset
+=== Aws sqs metricset
```

Comment:
AWS? Applies also to the others?

SQS? @kaiyan-sheng

Reply:
Changing AWS, but let's keep the metricset names lowercase for now.